tableauserverclient/models/revision_item.py

```diff
@@ -1,46 +1,66 @@
 import xml.etree.ElementTree as ET
+from ..datetime_helpers import parse_datetime
+from typing import List, Optional, TYPE_CHECKING, Type
+
+if TYPE_CHECKING:
+    from datetime import datetime
 
 class RevisionItem(object):
     def __init__(self):
-        self._resource_id = None
-        self._resource_name = None
-        self._revision_number = None
-        self._current = None
-        self._deleted = None
-        self._created_at = None
+        self._resource_id: Optional[str] = None
+        self._resource_name: Optional[str] = None
+        self._revision_number: Optional[str] = None
+        self._current: Optional[bool] = None
+        self._deleted: Optional[bool] = None
+        self._created_at: Optional["datetime"] = None
+        self._user_id: Optional[str] = None
+        self._user_name: Optional[str] = None
 
     @property
-    def resource_id(self):
+    def resource_id(self) -> Optional[str]:
         return self._resource_id
 
     @property
-    def resource_name(self):
+    def resource_name(self) -> Optional[str]:
         return self._resource_name
 
     @property
-    def revision_number(self):
+    def revision_number(self) -> Optional[str]:
         return self._revision_number
 
     @property
-    def current(self):
+    def current(self) -> Optional[bool]:
         return self._current
 
     @property
-    def deleted(self):
+    def deleted(self) -> Optional[bool]:
         return self._deleted
 
     @property
-    def created_at(self):
+    def created_at(self) -> Optional["datetime"]:
         return self._created_at
 
+    @property
+    def user_id(self) -> Optional[str]:
+        return self._user_id
+
+    @property
+    def user_name(self) -> Optional[str]:
+        return self._user_name
+
     def __repr__(self):
         return (
             "<RevisionItem# revisionNumber={_revision_number} "
-            "current={_current} deleted={_deleted}>".format(**self.__dict__)
+            "current={_current} deleted={_deleted} user={_user_id}>".format(**self.__dict__)
         )
 
     @classmethod
-    def from_response(cls, resp, ns, resource_item):
+    def from_response(
+        cls: Type["RevisionItem"],
```

Comment:
nit: I don't think we need to annotate the `cls` here, since it is passed in by the language itself and not the user

Reply:
Removed the type annotation.